CODEOWNERS

```diff
@@ -61,7 +61,7 @@ Kconfig*                                  @SebastianBoe
 /lib/lte_link_control/                    @jtguggedal
 /lib/modem_info/                          @rlubos
 /lib/modem_key_mgmt/                      @rlubos
-/lib/pdn_management/                      @rlubos
```

Comment:
Actually since @rlubos was the owner of the previous lib, you can remove myself and keep him instead.

Reply:
ok, will change.